lib/util/tooltip.coffee

```diff
@@ -31,9 +31,13 @@ class Tooltip
     @showOnHover()
     this
 
+  hide_: ->
```

Comment:
I just wanted a way to hide this immediately. Otherwise it's janky if you are hovering while all progress bars end.

The function thing is my screwup though.